lib/annotations/example.js

```diff
@@ -121,9 +147,16 @@ module.exports = (env) => {
               sassPromises.push(promise);
             }
           }
-          const iframePromise = Promise.all(sassPromises).then(() =>
-            renderIframe(env, exampleItem, 'example'),
-          );
+          const iframePromise = Promise.all(sassPromises)
+            .then(() => {
```

Comment:
Should this be `finally` instead of `then` so that the compiler is also disposed if there's an error?